website/docs/faqs/Git/git-migration.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "How to migrate git providers"
+sidebar_label: "Git Migration"
+id: "git-migration"
+description: "Learn how to migrate git providers in dbt Cloud with minimal disruption."
+tags: [Git]
+---
+
+To migrate from one git provider to another, refer to the following steps to avoid minimal disruption:
+
+1. Outside of dbt Cloud, you'll need to import your existing repo into your new provider. 
+   
+   As an example, if you're migrating from GitHub to Azure DevOps, you'll need to import your existing repo (GitHub) into your new git provider (Azure DevOps). For detailed steps on how to do this, refer to your git provider's documentation (Such as [GitHub](https://docs.github.com/en/migrations/importing-source-code/using-github-importer/importing-a-repository-with-github-importer), [GitLab](https://docs.gitlab.com/ee/user/project/import/repo_by_url.html), [Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/repos/git/import-git-repository?view=azure-devops)) 
+   
+2. Go back to dbt Cloud and set up your [integration for the new git provider](/docs/cloud/git/connect-github), if needed. 
```

Comment:
@b-per , not sure if step 2 is the same as step 4 - is step 2 needed IF they don't have an integration set up in dbt cloud already? or are we asking the users to integrate with the new git provider twice?

Reply:
Steps 2 and 4 are definitely different, so it looks good to me.

Step 2 is about setting up the integration (at most required once per account) whereas step 4 is connection the dbt Cloud project to the repository (required for each dbt Cloud project, so potentially more than once)